src/jquery.multiselect.js

```diff
@@ -616,12 +619,12 @@
       this._trigger('close');
     },
 
-    enable: function() {
-      this._toggleDisabled(false);
+    enable: function(toggleInputs) {
+      this._toggleDisabled(false, toggleInputs === undefined ? true : toggleInputs);
```

Comment:
It might be easier (more readable) to just check the parameter in the function instead of using a ternary operator in the function call.